tests/integration_tests/performance/test_snapshot_restore_performance.py

```diff
@@ -197,14 +169,12 @@ def consume_output(cons, latencies):
         (12288, 7),
     ],
 )
-def test_snapshot_scaling(
-    bin_cloner_path, microvm_factory, rootfs, guest_kernel, st_core, mem, vcpus
+def test_snapshot_scaling_vcpus(
+    microvm_factory, rootfs, guest_kernel, st_core, mem, vcpus
```

Comment:
Is this supposed to be renamed, or is it a rebase artifact?

Reply:
Rebase artifact... and I was thinking I was getting competent at rebasing :rofl:

Reply:
Tale as old as time :joy: